accessibility-checker-engine/help/WCAG20_Html_HasLang.mdx

```diff
@@ -31,8 +31,10 @@ Page must identify the default language of the document with a `lang` attribute
 
 ## What to do
 
-* Identify the default language of the document by providing the `lang` or `xml:lang` attribute on the HTML element.
-For example: `<html lang="en">`
+* Identify the default language of the HTML document by providing the `lang` attribute on the HTML element (e.g. `<html lang="en">`);
+* OR, if serving an XML document only (ie. using a MIME type such as application/xhtml+xml), identify the default language by providing the xml:lang attribute alone. The lang attribute is not needed (`<html xml:lang="fr" xmlns="http://www.w3.org/1999/xhtml">`)
+* OR, if serving XHTML 1.x or polyglot pages as text/html, use both the lang attribute and the xml:lang attribute and ensure the values for both atttributes are identical (`<html lang="fr" xml:lang="fr" xmlns="http://www.w3.org/1999/xhtml">`)
```

Comment:
Add `` for attribute names:

* OR, if serving XHTML 1.x or polyglot pages as text/html, use both the `lang` attribute and the `xml:lang` attribute and ensure the values for both atttributes are identical (`<html lang="fr" xml:lang="fr" xmlns="http://www.w3.org/1999/xhtml">`)

Reply:
Done, thanks